jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -528,9 +528,17 @@ def _calc_background_median(self, reg, data=None):
                 data = self.dataset.selected_dc_item
 
         comp = data.get_component(data.main_components[0])
+        if self.config == "cubeviz" and data.ndim > 2:

```

Comment:
Is it safe to generalize this?  Or do we worry about a case where the extra axis isn't spectral (maybe we can check directly for that instead)?

```suggestion
        if data.ndim > 2:
```